smarts/core/smarts.py

```diff
@@ -340,10 +340,32 @@ def _setup_bullet_client(self, client: bc.BulletClient):
         client.setGravity(0, 0, -9.8)
 
         plane_path = self._scenario.plane_filepath
+
+        bounding_box_lenght = (
```

Comment:
Maybe move the size and center calculation to scenario.py?

Reply:
spelling `bounding_box_lenght` --> `bounding_box_length`